membersrvc/ca/ca.go

```diff
@@ -340,6 +345,9 @@ func (ca *CA) createCertificate(id string, pub interface{}, usage x509.KeyUsage,
 }
 
 func (ca *CA) createCertificateFromSpec(spec *CertificateSpec, timestamp int64, kdfKey []byte) ([]byte, error) {
+	mutex.Lock()
```

Comment:
@jeronimoirazabal Yes, it makes sense :) I commented because I'm in general worried about locks. When they are not really needed they just reduce performances.